src/core/mod.rs

```diff
@@ -159,18 +202,20 @@ pub fn check_fault_injection(fault_name: &str) -> bool {
 }
 
 #[repr(u32)]
-#[derive(Debug, Clone, Eq, PartialEq, PartialOrd, Ord, Hash, Copy)]
+#[derive(Debug, Clone, Eq, PartialEq, PartialOrd, Ord, Hash, Copy, Deserialize)]
 pub enum StacksEpochId {
-    Epoch10 = 0x0100,
-    Epoch20 = 0x0200,
-    Epoch21 = 0x0201,
+    Epoch10 = 0x01000,
+    Epoch20 = 0x02000,
+    Epoch2_05 = 0x02005,
+    Epoch21 = 0x02010,
```

Comment:
This should be `0x0200a`.  `0x0a` is 10, not `0x10`.

Reply:
yikes